# Boolean variable names need to start with `is` (`wittgenstein/variable-name-boolean`)

ðŸ’¡ This rule is manually fixable by [editor suggestions](https://eslint.org/docs/developer-guide/working-with-rules#providing-suggestions).

<!-- end auto-generated rule header -->

## Rule Details

This rule aims to make sure you boolean variable name is start with `is`.

Examples of **incorrect** code for this rule:

```js
const show = true;
```

Examples of **correct** code for this rule:

```js
const isShow = true;
```
